salt/modules/consul.py

```diff
@@ -2126,7 +2131,7 @@ def acl_delete(consul_url=None, **kwargs):
         ret['res'] = False
         return ret
 
-    function = 'acl/delete/{0}'.format(kwargs['id'])
+    function = 'acl/destroy/{0}'.format(kwargs['id'])
```

Comment:
I can change that, but is this really necessary as i can't find when this API was "delete" instead of "destroy", does this function ever work ?